mlrun/projects/project.py

```diff
@@ -2011,14 +2037,17 @@ def clear_context(self):
         ):
             shutil.rmtree(self.spec.context)
 
-    def save(self, filepath=None):
+    def save(self, filepath=None, store=True):
         self.export(filepath)
-        self.save_to_db()
+        self.save_to_db(store)
```

Comment:
This is a customer facing function please add description about the `store` param